core/src/main/java/hudson/Functions.java

```diff
@@ -1936,6 +1936,17 @@ public boolean hyperlinkMatchesCurrentPage(String href) {
         return url.endsWith(href);
     }
 
+    /**
+     * Returns an immutable list containing only the specified object.
+     * The returned list is serializable.
+     *
+     * @param <T> The class of the objects in the list.
+     * @param t The sole object to be stored in the returned list.
+     * @return An immutable list containing only the specified object.
+     * @since 1.312
+     * @deprecated use {@link List#of}
+     */
+    @Deprecated(forRemoval = true, since = "TODO")
```

Comment:
Not sure about this one. `Functions` is the UI support class (it's bound to `h` in Jelly), and there's no convenient replacement for callers like https://github.com/jenkinsci/jenkins/blob/d41389131bed2083ffb02c129cced3b605bef02b/core/src/main/resources/hudson/model/Computer/sidepanel.jelly#L49

Reply:
> there's no convenient replacement for callers like

Sure there is. I migrated to existing callers in 8cfadaf28d. A quick code search showed there are only 11 others:

* https://github.com/jenkinsci/depgraph-view-plugin/blob/626a9068b4c8971a8b8cac85847e4b6d3497991a/src/main/resources/hudson/plugins/depgraph_view/AbstractDependencyGraphAction/sidepanel.jelly
* https://github.com/jenkinsci/deployer-framework-plugin/blob/1d1888e8c0214fa42ee932b3eb26c06a6c7279f0/src/main/resources/com/cloudbees/plugins/deployer/DeployNowComputer/sidepanel.jelly
* https://github.com/jenkinsci/emailext-template-plugin/blob/0b3354ceb081b070cc0eb204262307b5c6dda333/src/main/resources/org/jenkinsci/plugins/emailext_template/ExtEmailTemplateManagement/sidepanel.groovy
* https://github.com/jenkinsci/htmlresource-plugin/blob/a10c444cc158ec1413a7095800e22096981da43f/src/main/resources/com/cwctravel/jenkinsci/plugins/htmlresource/HTMLResourceManagement/sidepanel.jelly
* https://github.com/jenkinsci/implied-labels-plugin/blob/cb5bb29725c123b19f1c12d18286842e4dc49443/src/main/resources/org/jenkinsci/plugins/impliedlabels/Config/configure.jelly
* https://github.com/jenkinsci/implied-labels-plugin/blob/cb5bb29725c123b19f1c12d18286842e4dc49443/src/main/resources/org/jenkinsci/plugins/impliedlabels/Config/index.jelly
* https://github.com/jenkinsci/openstack-cloud-plugin/blob/4830e314d47c967a2f0825f563e171b9fda61e47/plugin/src/main/resources/jenkins/plugins/openstack/compute/JCloudsComputer/sidepanel.jelly
* https://github.com/jenkinsci/puppet-enterprise-pipeline-plugin/blob/ee775cb9b27bbaae2a7cb9476b4862bf4ea32fcc/src/main/resources/org/jenkinsci/plugins/puppetenterprise/api/HieraDataStore/sidepanel.jelly
* https://github.com/jenkinsci/puppet-enterprise-pipeline-plugin/blob/ee775cb9b27bbaae2a7cb9476b4862bf4ea32fcc/src/main/resources/org/jenkinsci/plugins/puppetenterprise/PuppetEnterpriseManagement/sidepanel.jelly
* https://github.com/jenkinsci/scriptler-plugin/blob/52a1e4bdc537f2757dd9d3f8c923e2225bc6ecf1/src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/sidepanel.jelly
* https://github.com/jenkinsci/shared-objects-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/sharedobjects/SharedObjectManagementResult/sidepanel.jelly

Reply:
I think we have very different ideas what's _convenient_.

There are _many_ methods in this class that basically wrap a static call, I don't see what's particularly wrong with this one.

Reply:
I would also prefer to avoid `j:invokeStatic` wherever possible (https://issues.jenkins.io/browse/JENKINS-31203).

In this case https://commons.apache.org/proper/commons-jexl/reference/syntax.html claims that JEXL has a list literal syntax, so perhaps just

```xml
<t:executors computers="${[it]}"/>
```

works?